tests/js/client/server_permissions/test-foxx-apis-off.js

```diff
@@ -120,8 +117,7 @@ function testSuite() {
         routes.forEach(function(route) {
           let result = arango.GET("/_api/foxx/" + route);
           assertTrue(result.error);
-          assertEqual(403, result.code);
-          assertEqual(3099, result.errorNum);
+          assertEqual(404, result.code);
```

Comment:
doesn't this return an arango-error anymore? if it does, the value should be checked.